posts/flock_2024_diary/index.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Packit Flock 2024 Diary"
+date: 2024-09-05T15:16:18+02:00
+authors: flachman
+tags:
+  - fedora
+  - flock
+  - conference
+  - "2024"
+---
+
+# Flock Packit Diary Blog post
+
+It’s been a month since two Packit members (Laura and František) attended Flock, the annual Fedora conference, in Rochester. Let’s see some highlights from our journey There and Back Again. Sadly the travel there wasn’t as expected at all, but luckily the tornadoes did not stop us and we both made it to Rochester.
```

Comment:
we could link ourselves as well since other people mentioned are linked

Reply:
Ah, now I see. I exported markdown directly from gdoc and it used the mails. I would avoid emails to not cause spam for people. What about mastodon?

Reply:
I've used Mastodon where I was able to find one. Matrix account otherwise...

Reply:
sounds good, GitHub would be ok as well